src/world/area_mac/mac_00/npc/russ_and_thief.inc.c

```diff
@@ -327,13 +337,26 @@ EvtScript N(EVS_ItemPrompt_Documents) = {
                     Return
                 EndIf
             Else
+#if VERSION_JP
+                IfEq(GF_Unused_MAC_74, TRUE)
+                    Call(SpeakToPlayer, NPC_SELF, ANIM_RussT_Talk, ANIM_RussT_Idle, 0, MSG_MAC_Gate_JP_0011)
+                    Set(LVarA, 1)
+                    Return
+                EndIf
+#endif
             EndIf
         CaseEq(-1)
         CaseDefault
             IfEq(LVar0, ITEM_MYSTERY_NOTE)
                 Call(SpeakToPlayer, NPC_SELF, ANIM_RussT_Talk, ANIM_RussT_Idle, 0, MSG_MAC_Gate_000E)
                 Set(GF_MAC00_TranslatedMysteryNote, TRUE)
                 Set(LVarA, 1)
+#if VERSION_JP
+            Else
+                Call(SpeakToPlayer, NPC_SELF, ANIM_RussT_Talk, ANIM_RussT_Idle, 0, MSG_MAC_Gate_0121)
+                Set(GF_Unused_MAC_74, TRUE)
```

Comment:
Worth looking at translating the message MAC_Gate_0121 so we can name this gameflag. It certainly shouldn't be marked unused

Reply:
I believe `GF_MAC00_TranslatedSuspiciousNote` works well. Here are the messages associated with it:

- MAC_Gate_JP_0011
In the Shy Guys' note you bought it was written that the guard's favorite food is "Lemon Candy".

- MAC_GATE_0121
Do you want me to read this note?
Errr...
It's written in Shy Guy language.
The contents are...
It's written that the guard's favorite food is "Lemon Candy".
What is this about?
Do you understand it?

- MAC_GATE_0122
Do you want me to read this note?
Hmmm...
It seems to be written in Shy Guy language...
If I had my dictionary I'd be able to read it but it was taken by the Shy Guys...
For the time being, I'll hold this note for you.

Reply:
Shounds good!